sdk/keyvault/azure-security-keyvault-administration/CMakeLists.txt

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+# setting CMAKE_TOOLCHAIN_FILE must happen before creating the project
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake-modules")
+include(AzureVcpkg)
+az_vcpkg_integrate()
+
+cmake_minimum_required (VERSION 3.13)
+project(azure-security-keyvault-administration LANGUAGES CXX)
+
+# Compile Options
+option(FETCH_SOURCE_DEPS "build source dependencies" OFF)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+if(FETCH_SOURCE_DEPS)
+    set(AZ_ALL_LIBRARIES ON)
+    include(FolderList)
+    SetCompileOptions(ADMINISTRATION)
+endif()
+
+include(AzureVersion)
+include(AzureCodeCoverage)
+include(AzureTransportAdapters)
+include(AzureDoxygen)
+include(AzureGlobalCompileOptions)
+include(AzureConfigRTTI)
+include(AzureBuildTargetForCI)
+# Add create_map_file function
+include(CreateMapFile)
+
+if(FETCH_SOURCE_DEPS)
+    GetFolderList(ADMINISTRATION)
+    foreach(oneFolder IN LISTS BUILD_FOLDERS)
+        message("add folder ${oneFolder}")
+        add_subdirectory(${oneFolder})
+    endforeach()
+elseif(NOT AZ_ALL_LIBRARIES)
+  find_package(azure-core-cpp "1.5.0" CONFIG QUIET)
+  if(NOT azure-core-cpp_FOUND)
+    find_package(azure-core-cpp "1.5.0" REQUIRED)
+  endif()
+endif()
+
+set(
+  AZURE_SECURITY_KEYVAULT_ADMINISTRATION_HEADER
+    inc/azure/keyvault/administration/dll_import_export.hpp
+    inc/azure/keyvault/administration/rtti.hpp
+    inc/azure/keyvault/administration/settings_client.hpp
+    inc/azure/keyvault/administration/settings_client_options.hpp
+    inc/azure/keyvault/administration/rest_client.hpp
+    inc/azure/keyvault/administration.hpp
```

Comment:
This should be the first one in the list

Reply:
done